File: Packet/ServerFull.cs
using System.IO;

namespace Resources.Packet {
    public class ServerFull : Packet {

        public ServerFull() : base() {
            PacketID = PacketID.Time;
        }
        public ServerFull(BinaryReader reader) : this() {
        }

        protected override void WritePacketData(BinaryWriter writer) { }
    }
}

File: src/main/kotlin/packets/ServerFull.kt
package com.github.lastexceed.cubeworldnetworking.packets

import com.github.lastexceed.cubeworldnetworking.utils.*

class ServerFull() : Packet(PacketId.ServerFull) {
	override suspend fun writeTo(writer: Writer) {}


	companion object : CwDeserializer<ServerFull> {
		override suspend fun readFrom(reader: Reader) = ServerFull()
	}
}